internal/cmd/config.go

```diff
@@ -503,22 +503,30 @@ func Configure(v *viper.Viper, _ *pflag.FlagSet) {
 
 	v.SetDefault("cluster::backyards::enabled", true)
 	v.SetDefault("cluster::backyards::istio::grafanaDashboardLocation", "./etc/dashboards/istio")
-	v.SetDefault("cluster::backyards::istio::pilotImage", "banzaicloud/istio-pilot:1.3.4-bzc")
-	v.SetDefault("cluster::backyards::istio::mixerImage", "banzaicloud/istio-mixer:1.3.4-bzc")
+	v.SetDefault("cluster::backyards::istio::pilotImage", "banzaicloud/istio-pilot:1.4.2-bzc")
+	v.SetDefault("cluster::backyards::istio::mixerImage", "banzaicloud/istio-mixer:1.4.2-bzc")
 	v.SetDefault("cluster::backyards::charts::istioOperator::chart", "banzaicloud-stable/istio-operator")
-	v.SetDefault("cluster::backyards::charts::istioOperator::version", "0.0.24")
+	v.SetDefault("cluster::backyards::charts::istioOperator::version", "0.0.32")
 	v.SetDefault("cluster::backyards::charts::istioOperator::values", map[string]interface{}{
 		"image": map[string]interface{}{
 			"repository": "banzaicloud/istio-operator",
-			"tag":        "0.3.5",
+			"tag":        "0.4.6",
 		},
 	})
 	v.SetDefault("cluster::backyards::charts::backyards::chart", "banzaicloud-stable/backyards")
-	v.SetDefault("cluster::backyards::charts::backyards::version", "1.0.4")
+	v.SetDefault("cluster::backyards::charts::backyards::version", "1.1.0")
 	v.SetDefault("cluster::backyards::charts::backyards::values", map[string]interface{}{
-		"image": map[string]interface{}{
-			"repository": "banzaicloud/backyards",
-			"tag":        "1.0.4",
+		"application": map[string]interface{}{
+			"image": map[string]interface{}{
+				"repository": "banzaicloud/backyards",
+				"tag":        "1.1.2",
+			},
+		},
+		"web": map[string]interface{}{
+			"image": map[string]interface{}{
+				"repository": "banzaicloud/backyards",
```

Comment:
Isn't this repo `backyards-web`?
```suggestion
				"repository": "banzaicloud/backyards-web",
```

Reply:
good catch, i fixed it, it wasn't properly used in the code